examples/layer-browser/src/examples/sample-layers.js

```diff
@@ -10,7 +10,8 @@ const S2LayerExample = {
     opacity: 0.6,
     getS2Token: f => f.token,
     getPath: f => f.path,
-    getFillColor: f => [f.value * 256, (1 - f.value) * 256, (1 - f.value) * 128],
+    getStrokeColor: f => [255, 255, 128],
```

Comment:
>     getStrokeColor: f => [255, 255, 128],
>     getFillColor: f => [255, 255, 128],

When color-picker.js calls these methods, it doesn't pass any parameters, `f` is going to be undefined, kept them in same style as `getStrokeWidth` below, but may be we should just change them to `callback: () => [255, 255, 128]` ?